instrumentation/netty/netty-4.0/src/main/java/io/opentelemetry/javaagent/instrumentation/hypertrace/netty/v4_0/NettyChannelPipelineInstrumentation.java

```diff
@@ -82,7 +82,7 @@ public static void addHandler(
         @Advice.Enter int callDepth,
         @Advice.This ChannelPipeline pipeline,
         @Advice.Argument(2) ChannelHandler handler) {
-      if (callDepth > 0) {
```

Comment:
Why do we want to increase the call depth here?

Reply:
This was an odd one, but I found this was  a niche bug I found where sometimes the handler's call depth was one but the channel pipeline had not yet had its tracing handlers replaced

Reply:
Shashank and I talked offline about this one - we only want to change this number for netty 4.1 due to the different way the call depth map is constructed